projects/templates/src/lib/components/po-page-dynamic-search/po-advanced-filter/po-advanced-filter-base.component.ts

```diff
@@ -51,7 +76,42 @@ export class PoAdvancedFilterBaseComponent {
     return this._filters;
   }
 
+  /** Objeto com as literais usadas no `po-advanced-filter`. */
+  @Input('p-literals') set literals(value: PoAdvancedFilterLiterals) {
+    if (value instanceof Object && !(value instanceof Array)) {
+      this._literals = {
+        ...poAdvancedFiltersLiteralsDefault[poLocaleDefault],
+        ...poAdvancedFiltersLiteralsDefault[this.language],
+        ...value
+      };
+    } else {
+      this._literals = poAdvancedFiltersLiteralsDefault[this.language];
+    }
+
+    this.primaryAction.label = this.literals.confirmLabel;
+    this.secondaryAction.label = this.literals.cancelLabel;
+  }
+
+  get literals() {
+    return this._literals || poAdvancedFiltersLiteralsDefault[this.language];
+  }
+
   /** Função que será disparada e receberá os valores do formulário ao ser clicado no botão buscar. */
   @Output('p-search-event') searchEvent = new EventEmitter<any>();
 
+  constructor(languageService: PoLanguageService) {
+    this.language = languageService.getShortLanguage();
+  }
+
+    // Retorna os models dos campos preenchidos
+  private getValuesFromForm() {
+    Object.keys(this.filter).forEach(property => {
+      if (this.filter[property] === undefined || this.filter[property] === '') {
```

Comment:
Daria para simplificar para  Object.keys(this.filter).forEach(key => !this.filter[key] && delete this.filter[key]);
Oque vc acha?